front/app/containers/Admin/projects/project/events/edit.tsx

```diff
@@ -671,7 +619,7 @@ const AdminProjectEventEdit = () => {
                 </SectionField>
               )}
 
```

Comment:
(Unrelated to your PR, but using our new fontWeight semi-bold option, instead of inline styles. Same result.)